src/pudl/transform/ferc714.py

```diff
@@ -293,6 +317,26 @@ def _assign_respondent_id_ferc714(
     return df
 
 
+def _fillna_respondent_id_ferc714_source(
```

Comment:
this is actually also necessary in the hourly table as well!

Reply:
Worth checking whether it's necessary for the hourly table

Reply:
I think it will be nice to add them in to that table as well!